app/controllers/enrollments_controller.rb

```diff
@@ -104,8 +104,7 @@ def set_enrollment
 
   def enrollment_class
     type = params.fetch(:enrollment, {})[:fournisseur_de_donnees]
-    type = %w[dgfip api-particulier api-entreprise].include?(type) ? type : nil
-
+    type = %w[dgfip api-particulier api-entreprise franceconnect].include?(type) ? type : nil
```

Comment:
There is `api-entreprise` here, how come ? I did not see it in Oauth or front. However, `cnam` is in front and oauth, but not here...

Reply:
see https://trello.com/c/0Dl3G5gJ/157-refacto-linterface-oauth2-de-signup-back-suit-une-impl%C3%A9mentation-standard